syserol/figures/figureS1.py

```diff
@@ -22,7 +22,7 @@ def makeFigure():
         ax[i].set_ylabel("Predicted Values")
         m, b = np.polyfit(x, y, 1)  # line of best fit
         ax[i].plot(x, m * x + b, 'k--', color="red")
-        ax[i].text(1, 1, f"Accuracy Score: {round(accuracy, 3)}", {"color": "red", "fontsize": 10}, horizontalalignment="right",
```

Comment:
I adjusted this. Please check its placement before merging.

Reply:
Looks good to me, I'll merge now